### PR TITLE
Some changes here and there

### PR DESCRIPTION
**Pull request by Regular-Baf**

### CODE REVIEW

File: src/wiki/development/development-builds.md
Comment (DioEgizio) [inline src/wiki/development/development-builds.md:11]: Actually, Linux portable builds are back

File: src/wiki/development/development-builds.md
Comment (DioEgizio) [inline src/wiki/development/development-builds.md:8]:
```suggestion
Please understand that **these builds are not intended for most users**. There may be bugs, and other instabilities. You have been warned.
```

**Review by DioEgizio — CHANGES_REQUESTED** [review]
Other than these, LGTM

File: src/wiki/development/development-builds.md
Comment (DioEgizio) [reply src/wiki/development/development-builds.md:11]: Also MacOS is *not* portable, nor is AppImags

**Comment by ZekeSmith** [comment]
This is looking pretty good I am quite glad someone is able to fix my terrible English. Have a look at my review and @DioEgizio 's after that it should be good to merge.

File: src/wiki/development/development-builds.md
Comment (Regular-Baf) [reply src/wiki/development/development-builds.md:11]: I was just going off of what was there before. Are you sure about AppImage though, I thought being portable was the point of them?

File: src/wiki/development/development-builds.md
Comment (Regular-Baf) [reply src/wiki/development/development-builds.md]: thanks for catching that one

File: src/wiki/development/development-builds.md
Comment (DioEgizio) [reply src/wiki/development/development-builds.md:11]: AppImage is not portable since it saves data in home directories

Comment (Regular-Baf) [reply src/wiki/development/development-builds.md:11]: I'll clean that one up then, thank you

**Comment by Regular-Baf** [comment]
How's this looking now?

**Comment by DioEgizio** [comment]
> How's this looking now?

>Portable dev-builds are, as of now, only provided for Windows.

As I said, Linux portable builds are back

File: src/wiki/development/development-builds.md
Comment (DioEgizio) [inline src/wiki/development/development-builds.md:11]:
```suggestion
Portable dev-builds are provided for Windows and Linux, while there's also a binary Linux tarball, an appimage and a MacOS .app
```

Comment (DioEgizio) [reply src/wiki/development/development-builds.md]: @Regular-Baf apply that

Comment (Regular-Baf) [reply src/wiki/development/development-builds.md]: for some reason it won't let me, the commit button is greyed out.
I'll type it in

**Review by DioEgizio — APPROVED** [review]
Lgtm

File: src/wiki/installing/windows.md
Comment (DioEgizio) [inline src/wiki/installing/windows.md:40]:
```suggestion
If you want a more technical explanation, it's because msys2 and Cygwin, that we use for building PolyMC on Windows, will start targeting 8.1 ([see this](https://www.msys2.org/docs/windows_support/)).
```

Comment (Regular-Baf) [reply src/wiki/installing/windows.md:40]: yeah I wasn't too sure about that one, thank you

**Comment by ZekeSmith** [comment]
looks good

**Pull request merged**
